proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -38,6 +38,14 @@ message EventCreateBatch {
   // end_date is the end of the period during which this credit batch was
   // quantified and verified.
   string end_date = 6 [ (gogoproto.moretags) = "yaml:\"end_date\"" ];
+
+  // project_location is the location of the project backing the credits in this
+  // batch. It is a string of the form
+  // <country-code>[-<sub-national-code>[ <postal-code>]], with the first two
+  // fields conforming to ISO 3166-2, and postal-code being up to 64
+  // alphanumeric characters. country-code is required, while sub-national-code
+  // and postal-code can be added for increasing precision.
```

Comment:
Please document the `sub-national-code` format. Is there a limit of characters?

Reply:
It's documented in ISO 3166-2